src/Expensify.js

```diff
@@ -81,6 +89,7 @@ const defaultProps = {
     updateAvailable: false,
     isSidebarLoaded: false,
     screenShareRequest: null,
+    isCheckingPublicRoom: true,
```

Comment:
Isn't this gonna slow down all app initializations even when users already have a report stored locally?

Reply:
Ah, this only applies when there's no authToken and when the app is opened from a link containing a reportID. So in that case we just keep showing the splash screen until we know if the server responds. Otherwise, we just set this flag to false.

Reply:
Where do we check for the absence of authToken?